redis/Package.md

```diff
@@ -1,120 +1,21 @@
-## Module Overview
+Connects to Redis from Ballerina
```

Comment:
Suggestion: "This module connects to Redis from Ballerina."

Reply:
This is how it is in common format

Reply:
Connector name is automatically rendered at Ballerina Central. So we skip the connector name.